### PR TITLE
Replace colons with a period

### PR DESCRIPTION
**Pull request by sunithasomanna**
Replacing colons with period for relevant parameters in the API readme.
Also removed extra spaces before the headings

### CODE REVIEW

**Review by rohitkumar-ro — APPROVED** [review]
Changes looks fine

**Review by arahimanshaik — APPROVED** [review]
Looks good for me

**Review by hemkapoor — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -182,38 +182,29 @@
...
-  The plugins abstract, translate, and expose southbound resource information to the resource aggregator through
-  RESTful APIs. HPE Resource Aggregator for ODIM supports:
-
-    - Generic Redfish plugin for ODIM (GRF): Generic Redfish plugin that can be used as a plugin for any Redfishcompliant
-      device.
-	- Plugin for unmanaged racks (URP): Plugin that acts as a resource manager for unmanaged racks. 
+  The plugins abstract, translate, and expose southbound resource information to the resource aggregator through RESTful APIs. HPE Resource Aggregator for ODIM supports:
+  
+  - Generic Redfish plugin for ODIM (GRF): Generic Redfish plugin that can be used as a plugin for any Redfish-compliant device.
+  - Plugin for unmanaged racks (URP): Plugin that acts as a resource manager for unmanaged racks.
```
Comment (Bharath-KKB) [inline docs/README.md:203]: We are missing Cisco ACI, Lenovo and Dell plugins?

File: docs/README.md [hunk 2]
```diff
@@ -182,38 +182,29 @@
...
+  - Plugin for unmanaged racks (URP): Plugin that acts as a resource manager for unmanaged racks. 
     - Integration of additional third-party plugins.  	
 
-This guide provides reference information for the northbound APIs exposed by the resource aggregator. These APIs
-are designed as per DMTF's [Redfish® Scalable Platforms API (Redfish) specification 1.11.1](https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.11.1.pdf) and are fully Redfish-compliant.
-Redfish® is an open industry standard specification, API, and schema. It specifies a RESTful interface and uses JSON and OData. The Redfish
-standards are designed to deliver simple and secure environment for managing multivendor, converged, and hybrid IT infrastructure.
+This guide provides reference information for the northbound APIs exposed by the resource aggregator. These APIs are designed as per DMTF's [Redfish® Scalable Platforms API (Redfish) specification 1.11.1](https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.11.1.pdf) and are fully Redfish-compliant.
```
Comment (Bharath-KKB) [inline docs/README.md:206]: Confirm if this is the right version

File: docs/README.md [hunk 3]
```diff
@@ -227,48 +218,31 @@ The following figure shows these functional layers of Resource Aggregator for OD
...
-This layer hosts a REST server which is open-source and secure. It learns about the southbound resources from
-the plugin layer and exposes the corresponding Redfish data model payloads to the northbound clients. The
-northbound clients communicate with this layer through a REST-based protocol that is fully compliant with
-DMTF's Redfish® specifications (Schema 2020.3 and Specification 1.11.1).
+This layer hosts a REST server which is open-source and secure. It learns about the southbound resources from the plugin layer and exposes the corresponding Redfish data model payloads to the northbound clients. The northbound clients communicate with this layer through a REST-based protocol that is fully compliant with DMTF's Redfish® specifications (Schema 2020.3 and Specification 1.11.1).
 The API layer sends user requests to the plugins through the aggregation, the event, and the fabric services.
 
 - **Services layer**
 
 The services layer is where all the services are hosted. This layer implements service logic for all use cases
```
Comment (Bharath-KKB) [inline docs/README.md:226]: This is wrong. Services and collections both provide functionality. Also the figure is that of collections... no services there

File: docs/README.md [hunk 4]
```diff
@@ -227,48 +218,31 @@ The following figure shows these functional layers of Resource Aggregator for OD
...
 through an extensible domain model (Redfish Data Model). Requests coming from the API layer and the
-responses coming from the plugin layer are mapped to the actual end resources in this layer. It maintains the state
-for event subscriptions, credentials, and tasks. It also hosts a message bus called the Plug-in Message Bus (PMB).
+responses coming from the plugin layer are mapped to the actual end resources in this layer. It maintains the state for event subscriptions, credentials, and tasks. It also hosts a message bus called the Plug-in Message Bus (PMB).
```
Comment (Bharath-KKB) [inline docs/README.md:228]: What is a PMB?

Comment (Bharath-KKB) [reply docs/README.md:228]: Plus _" Requests coming from the API layer and the responses coming from the plugin layer are mapped to the actual end resources in this layer."_ is not correct.

Comment (Bharath-KKB) [reply docs/README.md:228]: This _"This layer hosts a message broker which acts as a communication channel between the upper layers and the plugin layer. It supports common messaging architecture and real-time streaming. Resource Aggregator for ODIM uses Kafka as the event message bus. The services and the event message bus layers host Redis data store."_ needs changes. This is only used for events. And arent we supporting redis streams ?

File: docs/README.md [hunk 5]
```diff
@@ -227,48 +218,31 @@ The following figure shows these functional layers of Resource Aggregator for OD
...
-v3.0 to interact with the other layers. It collects events to be exposed to fault management systems and uses the
-event message bus to publish events. The messaging mechanism is based on OpenMessaging Specification.
-The plugin layer allows developers to create plugins on any tool set of their choice without enforcing any strict
-language binding. To know how to develop plugins, refer to [Resource Aggregator for Open Distributed
-Infrastructure Management Plugin Developer's Guide](https://github.com/ODIM-Project/ODIM/blob/development/plugin-redfish/README.md).
+This layer connects the actual managed resources to the aggregator layers and is de-coupled from the upper layers. A plugin abstracts vendor-specific access protocols to a common interface which the aggregator layers use to communicate with the resources. It uses REST-based communication which is based on OpenAPI Specification v3.0 to interact with the other layers. It collects events to be exposed to fault management systems and uses the event message bus to publish events. The messaging mechanism is based on OpenMessaging Specification.
```
Comment (Bharath-KKB) [inline docs/README.md:239]: We cannot claim OpenAPI v3 support. Also _"The messaging mechanism is based on OpenMessaging Specification."_ doesnt look right

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
Pls fix the issues identified.

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md:203]: Addressed all your comments on a different PR (https://github.com/sunithasomanna/ODIM/pull/21). Please review the updated document at https://github.com/sunithasomanna/ODIM/blob/sprint-47-odim-68/docs/README.md

**Review by Bharath-KKB — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Comment by sunithasomanna** [comment]
Addressed all comments given by Bharath K, on a different PR (#21). Please review the updated document at https://github.com/sunithasomanna/ODIM/blob/sprint-47-odim-68/docs/README.md

**Pull request merged**